23-07-workshop/finished/Leavitts_Footsteps_solutions.ipynb

```diff
@@ -0,0 +1,850 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    for i in range(len(ticlist)):
</pre><p> </p><p>do we need "i"? we could do something like</p><p> </p><pre class="ql-syntax" spellcheck="false">for tic in ticlist:
&nbsp; &nbsp; # Get the obs, keep only the first matching
&nbsp; &nbsp; obs = lk.search_lightcurve(tic)
&nbsp; &nbsp; tesslc = obs[0].download()
&nbsp; &nbsp;&nbsp;
&nbsp; &nbsp; # Create the periodogram, find period
&nbsp; &nbsp; pgrm = tesslc.to_periodogram()
&nbsp; &nbsp; p_max = pgrm.period_at_max_power
&nbsp; &nbsp; periods.append(p_max.value)
&nbsp; &nbsp;&nbsp;
&nbsp; &nbsp; # Plot lc
&nbsp; &nbsp; tesslc.fold(p_max).scatter(label = f"TIC={tic}, Period = {p_max.value:.3f}d");
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>This just in! Lightkurve 2.4.1 now supports TGLC, so we could actually use this</p>